sql/src/main/java/io/crate/operation/collect/ShardCollectService.java

```diff
@@ -249,10 +234,22 @@ private CrateCollector getLuceneIndexCollector(ThreadPool threadPool,
     }
 
     public OrderedDocCollector getOrderedCollector(RoutedCollectPhase collectPhase,
-                                                SharedShardContext sharedShardContext,
-                                                JobCollectContext jobCollectContext) {
-        collectPhase = collectPhase.normalize(shardNormalizer);
+                                                   SharedShardContext sharedShardContext,
+                                                   JobCollectContext jobCollectContext) {
+        RoutedCollectPhase normalizedCollecPhase = collectPhase.normalize(shardNormalizer);
```

Comment:
Typo: CollecPhase instead of CollectPhase and actually can be moved inside the if block without local variable.